src/panel-app/apps.component.js

```diff
@@ -1,10 +1,13 @@
-import React, { useState, useEffect } from "react";
-import { Scoped, always, maybe } from "kremling";
+import React, { useState, useEffect, useMemo } from "react";
+import { Scoped, always } from "kremling";
```

Comment:
you actually use `maybe` below.

if you removed it because of VSCode, I've found that it almost always says you don't use `maybe` when you actually do. should probably report it as a bug or something.

Reply:
Oops. I did remove it because of VS code, though it doesn't break because I'm not using the bare identifier version. It's chained on the `always`, so it would have only broken if `maybe` was the first in the chain... maybe. I don't think this is a bug but I could be wrong.

Reply:
ahh, good point. the maybe that's used comes from the chain.